### PR TITLE
Fix wrong place for license scan badge in README.md (#192)

### PR DESCRIPTION
**Pull request by f7753**
Move the badge from inner chapter to file head.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Apache Pulsar manager is a web-based GUI management tool for managing and monitoring Pulsar.
 
+[![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fstreamnative%2Fpulsar-manager.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Fstreamnative%2Fpulsar-manager?ref=badge_shield)
```
Comment (tuteng) [inline README.md]: Thank you very much for your contribution.
What do you think of putting the badge at the top of the content, similar to the following
image

Comment (f7753) [reply README.md]: OK, I shall make another commit.
But should I put this badge below or beyond the chapter line of Apache Pulsar manager?
image

Comment (tuteng) [reply README.md]: I think this is better

image

Comment (f7753) [reply README.md]: Yep, agree with that.

**Pull request merged**